### PR TITLE
Remove dashboard functionality

### LINKED ISSUE #4863: Remove dashboard functionality
### Description

Lisk Dashboard was provides overview of recent transaction, news feed and balance. Except news feed all the features are available in `Wallet` tab. As of today we do not know the usability metrics of the news feed features and its benefits.

Both `Twitter` and `Lisk Blogs` are independently accessible with dedicated domain, we do not see much value add at the moment to keep it in Lisk Desktop mainly with following reasons:

- Development & Maintenance cost
- Operation costs

### Acceptance Criteria

- Remove dashboard and its features completely
  - Remove `Newsfeed`
  - Remove `Recent transactions widget`
  - Remove `Wallet details widget`
- Make `Wallet` tab as default
  - Update previous routes from `dashboard` to `wallet`
  - Update reference texts
- Update introduction [UI](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=3996-262974&t=nEALWKheKDSwPb82-4) to wallet (when users have no accounts managed in wallet) 
- Update the states, hooks, selectors used specifically for dashboard to wallet

### PR DESCRIPTION
**Pull request by oskarleonard**
### What was the problem?

This PR resolves #4863

### How was it solved?
- Removing dashboard and its components
- Replacing dashboard redirect logic to wallet
- Implementing new welcome GUI
- Removing dashboard links

### How was it tested?
Manually

### CODE REVIEW

File: src/modules/account/components/AccountDetails/WelcomeView/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            {t('Add account')}
+          </Link>
+        </div>
+      </header>
+      <footer className={styles.welcomeFooter}>
+        <h2>{t('Why do I need an account?')}</h2>
+        <div className={styles.accountUniqueValuePropositionListing}>
+          {accountUniqueValuePropositions.map((valueProposition, index) => (
+              <AccountUniqueValueProposition
+                key={index}
```
Comment (ManuGowda) [inline src/modules/account/components/AccountDetails/WelcomeView/index.js]: maybe start with index 1?

File: src/modules/auth/components/SetPasswordSuccess/SetPasswordSuccess.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ function SetPasswordSuccess({ onClose, encryptedPhrase, headerText, contentText,
         </p>
         <DownloadJSON fileName={fileName} encryptedPhrase={encryptedPhrase} />
         <PrimaryButton className={styles.continueButton} onClick={onContinue}>
-          {buttonText || t('Continue to dashboard')}
+          {buttonText || t('Continue to account')}
```
Comment (ManuGowda) [inline src/modules/auth/components/SetPasswordSuccess/SetPasswordSuccess.js]:
```suggestion
          {buttonText || t('Continue to wallet')}
```

Comment (ManuGowda) [reply src/modules/auth/components/SetPasswordSuccess/SetPasswordSuccess.js]: For now lets keep it wallet, we need to rethink `wallet vs accounts`, until then `wallet` will be our reference

**Review by ManuGowda — COMMENTED** [review]
Remove assets or other reference related to 
- dashboard
- newsfeed

e.g setup/react/assets/images/icons/news-feed-blog-dark.svg

Also fix the button text underline for both buttons

Screenshot 2023-03-31 at 11 40 00 AM

File: src/modules/common/components/NotFound/index.js [hunk 3]
```diff
@@ -13,11 +13,11 @@ const NotFound = ({ t }) => (
       <h2 className="empty-message">{t('Whoops, that page is gone.')}</h2>
       <p>
         {t(
-          'Sorry, we couldn’t find the page you were looking for. We suggest that you return to the main dashboard.'
+          'Sorry, we couldn’t find the page you were looking for. We suggest that you return to your account.'
```
Comment (ManuGowda) [inline src/modules/common/components/NotFound/index.js]:
```suggestion
          'Sorry, we couldn’t find the page you were looking for. We suggest that you return to your wallet.'
```

File: src/modules/common/components/NotFound/index.js [hunk 4]
```diff
@@ -13,11 +13,11 @@ const NotFound = ({ t }) => (
...
         )}
       </p>
-      <Link className="go-to-dashboard-button" to={routes.dashboard.path}>
-        <PrimaryButton className={styles.goToDashboardButton}>{t('Go to dashboard')}</PrimaryButton>
+      <Link className="go-to-dashboard-button" to={routes.wallet.path}>
```
Comment (ManuGowda) [inline src/modules/common/components/NotFound/index.js]: class names needs update?

**Review by ManuGowda — CHANGES_REQUESTED** [review]
In general please use `wallet` as replacement for `dashboard` for now 🙏🏼

File: src/modules/auth/components/SignupSuccessed/accountCreated.js [hunk 5]
```diff
@@ -17,7 +17,7 @@ const AccountCreated = ({ t }) => (
     <Illustration className={styles.illustration} name="registrationSuccess" />
     <p className={styles.subheader}>
       {t(
-        'You can now add your account to Lisk Wallet by clicking on "Continue to dashboard", to be able to send and request tokens and much more.'
+        'You can now add your account to Lisk Wallet by clicking on "Continue to account", to be able to send and request tokens and much more.'
```
Comment (ManuGowda) [inline src/modules/auth/components/SignupSuccessed/accountCreated.js]:
```suggestion
        'You can now add your account to Lisk Wallet by clicking on "Continue". Once your account is added to wallet you will be able to send and request tokens and do much more.'
```

**Review by ManuGowda — COMMENTED** [review]
Screenshot 2023-03-31 at 11 59 20 AM

When you remove all accounts, the modal show incorrect message `Continue to manage accounts`, i think just for this edge case we need to guide users stating, no accounts exists, and do it differently?

When the wallet doesn't have any accounts added, an action related to an address should be hidden, in this case, `My validator profile` or `Register validator` or `Stakes` should be hidden.

Screenshot 2023-03-31 at 12 05 25 PM

**Review by ManuGowda — DISMISSED** [review]

**Review by ManuGowda — DISMISSED** [review]

**Review by ManuGowda — DISMISSED** [review]

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]
## Review comments

- I guess this header should be sticky
image

- Figma says line height should be 20px but it seems to be lesser 
image

- Figma says font size should be 32px but its 30px
image

- color and font size seems to be wrong
**Figma says.**  
  - Font-size: 16px.   
  - color: #70778B
  
  **Actual says**
     - Font-size: 15px
     - color: rgb(0,0,0)
 
image
   

- Figma says the border-radius should be fully rounded but actual doesn't have a fully rounded border radius
image


- Figma says font size should be 20px but actual is 32px
image

**Review by eniolam1000752 — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**